force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -349,20 +438,28 @@ public with sharing class FlowControlLoopsCollections {
         // Initialize a Map
         Map<String, Integer> employeeSalaries = new Map<String, Integer>();
 
-        // Add key-value pairs to the Map
+        employeeSalaries.put('John Doe', 50000);// Add key-value pairs to the Map
         // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
+        employeeSalaries.put('Jane Smith', 60000);// add Jane Smith => 60000
+        employeeSalaries.put('Sam Brown', 55000);// add Sam Brown => 55000
+        employeeSalaries.put('Alice Johnson', 65000);// add Alice Johnson => 65000
 
+        List<String> highlyPaid = new List<String>();
+        
         // Initialize a list to store the names of high paid employees
+        for (String employeeName: employeeSalaries.keySet()){
+                Integer salary = employeeSalaries.get(employeeName);
+
 
-        // Iterate over the Map using a for loop
-            // Check if the salary of the employee is more than 55000
-                // Add the employee to the list of high paid employees
+            if (salary > 55000){
```

Comment:
Try creating a values for the number 55000 so it's easier to change in the future.